src/backend.c

```diff
@@ -137,13 +137,17 @@ SR_API GSList *sr_buildinfo_libs_get(void)
 		glib_binary_age, glib_interface_age));
 	l = g_slist_append(l, m);
 
+#ifdef CONF_ZLIB_VERSION
```

Comment:
These missing ifdefs caused me some issues when building the code, they really should be here